tests/TESTINS5.MLC

```diff
@@ -481,9 +490,13 @@ BIC_OK2  EQU    *
          RT1   CC2
          CLG   R2,=FL8'6'
          RT1    CC0 
+         ESPIE SET,ESPIE_EXIT,8,PARAM=MSGRKC_CC3
          LMG   R5,R6,=XL8'7FFFFFFFFFFFFFFF,2'
          SR       R0,R0
          MSGRKC  R2,R5,R6
+         RT1    ABORT
+MSGRKC_CC3 EQU *
+          WTO  'MSGRKC_CC3 OVERFLOW TEST OK'
           RT1    CC3  
           CLG   R2,=FL8'-2'  #300 CHK RESULT AFTER OVERFLOW
           RT1    CC0
```

Comment:
You should do ESPIE RESET, I think. The next ESPIE, in theory, does not replace this ESPIE, but apparently your testing proved otherwise. At any rate, in my mind it would be cleaner to reset it after each use.

Reply:
I now have added several commits included to add ESPIE RESET, move CC3 test ahead of WTO, and added more zero result tests.